script.js

```diff
@@ -349,7 +306,15 @@ for (let i = 0; i < myProjects.length; i += 1) {
     seeProjectButton(e.target.id);
   });
 }
-<<<<<<< HEAD
 
-=======
->>>>>>> f69eacf89ac65d382729aa20b2fbd3694dbe0859
+const form = document.getElementById("form-contact1");
+const emailInput = document.getElementById("user-email");
+const errorMessage = document.getElementById("error-message");
+
+form.addEventListener("submit", (e) => {
+  if (emailInput.value.toLowerCase() !== emailInput.value) {
+    e.preventDefault();
+    errorMessage.textContent =
+      "Please, use only lowercase letter for your email!";
+  }
+});
```

Comment:
- [ ] This change is not reflected in your project because you did not fix merge conflicts in the script.js file. Now your form does not validate the correct email format while I test it on my browser.

Reply:
GitHub said all checks passed, and I remember fixing this part, I will check again, thanks

Reply:
My VS Code says working on clean tree nothing to commit, and it isn't showing any merging conflicts, those were of the branch before, which I have corrected.

Reply:
No merging errors, it even says working on clean tree
![Sin título](https://user-images.githubusercontent.com/54015740/124051717-a6d8e280-d9e2-11eb-9aed-fba65d1fa24f.png)
I advise that when reviewing, please make sure, you are checking the right branch before sending your review because we have only 3 reviews per student.   []

Reply:
@sheylaPozo, I am so sorry. As you said I made a terrible mistake. I reviewed your code without checking out to the latest branch. I am sorry for my late reply too. I just have seen it now. :sob: I hope you already got another review and the TSE understood my mistake.